build.sh

```diff
@@ -2,53 +2,200 @@
 
 set -Eeou pipefail
 
-# color codes
+# fold me :) (in vscode vscodium etc)
```

Comment:
I don't really see the point of a "fold me" comment. Even more so a reference to one particular editor. I don't think this is helpful 

Frankly I find the region/end region comments more distracting than helpful. Please remove